dbml/gen_func.go

```diff
@@ -6,14 +6,14 @@ import (
 )
 
 type FuncInfo struct {
-	Name    string
-	Inputs  []string
-	Returns []string
-	Body    string
+	Name        string
+	Inputs      []string
+	Returns     []string
+	FinalReturn string
+	Body        string
 }
 
-func buildFuncInfo(fn DBMLFunc) (*FuncInfo, error) {
-
+func buildFuncInfo(fn DBMLFunc, hasDataPkg bool, pkg, dbName string) (*FuncInfo, error) {
```

Comment:
i wonder could these three fields combined into one string field named `externalDB`, and check if not empty string

Reply:
It's hard to combine them into one field as the selection of DB for further operations would be {pkg}.{dbName}.{funtcion}(e.g. data.GameClub.ExecContext()), it's better to split into 2 string field. I decided to delete bool field "hasDataPkg" and use "externalDB"(pervious field name - dbName) to check if it's an empty string. Would it be better?